inline.h

```diff
@@ -1133,6 +1133,12 @@ in lvalue context.
 */
 
 
+PERL_STATIC_INLINE U8 *
+Perl_bytes_to_utf8(pTHX_ const U8 *s, STRLEN *lenp)
+{
+    return bytes_to_utf8_free_me(s, lenp, NULL);
+}
+
```

Comment:
This should be above the doc comment (which is for the code following), and probably with other utf8 related functions.

Reply:
Indeed.  I wonder how it ended up there; all I can think of is during rebasing.